record/record_test.go

```diff
@@ -418,8 +421,19 @@ func TestComplexRecover(t *testing.T) {
 	buf[blockSize*3+2] = 0xad
 	buf[blockSize*3+3] = 0xde
 
-	// The first record should fail, but only when we read deeper beyond the
-	// first block.
+	// Now corrupt the two blocks in a row that correspond to records[2]
+	// and records[3].
+	buf[blockSize*4+0] = 0xef
+	buf[blockSize*4+1] = 0xbe
+	buf[blockSize*4+2] = 0xad
+	buf[blockSize*4+3] = 0xde
+	buf[blockSize*5+0] = 0xef
+	buf[blockSize*5+1] = 0xbe
+	buf[blockSize*5+2] = 0xad
+	buf[blockSize*5+3] = 0xde
```

Comment:
The "0xdeadbeef"s are neat when seen as a 32-bit value, but looking at it again, I think they're just confusing. Please change all of the RHS values to just "0".